lib/AuthStateManager.ts

```diff
@@ -155,6 +155,7 @@ export class AuthStateManager {
       };
 
       this._sdk.isAuthenticated()
+        .catch(() => {}) // Renew errors will emit an "error" event 
```

Comment:
seems like the fix for the issue should be in `isAuthenticated` which should return true or false and not throw an exception. As noted, renew errors will emit an "error" event, which is recommended way of seeing errors with auto renew (which can be triggered at any time in the "background").

Reply:
Changing isAuthenticated to not throw can introduce breaking change if any logic depends on thrown instead of emitted errors.

I agree isAuthenticated should not throw, but I would prefer to make the change in V6.0.